cluster/addons/cluster-monitoring/influxdb/heapster-controller.yaml

```diff
@@ -20,7 +20,7 @@ spec:
         kubernetes.io/cluster-service: "true"
     spec:
       containers:
-        - image: gcr.io/google_containers/heapster:v0.18.1
+        - image: thuctnguyen/heapster:v0.18.2-rc1
```

Comment:
This PR depends on the latest heapster code with support for InfluxDB 0.9.4. I built heapster from HEAD and created thuctnguyen/heapster:v0.18.2-rc1 from that binary. Is there a Docker image for the latest heapster code that I can use?